licenses/find.go

```diff
@@ -62,7 +63,7 @@ func findUpwards(dir string, r *regexp.Regexp, stopAt []*regexp.Regexp, predicat
 			return "", err
 		}
 		for _, f := range dirContents {
-			if r.MatchString(f.Name()) {
+			if r.MatchString(f.Name()) || r.MatchString(strings.ToUpper(f.Name())) {
```

Comment:
Could probably just do
```
if r.MatchString(strings.ToUpper(f.Name())) {
```
?

Reply:
I tried that, but the same function is used to find the .git directory, so either you need both original-case and uppercase options, or to parameterise or duplicate the `find-upwards` function